src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -102,11 +102,6 @@
 
         gb.withRateLimitHandler(CUSTOMIZED);
         OkHttpClient client = new OkHttpClient().setProxy(getProxy(defaultIfBlank(apiUrl, GITHUB_URL)));
-        client.setCache(GitHubClientCacheOps.toCacheDir().apply(config));
-        if (config.getClientCacheSize() > 0) {
-            Cache cache = toCacheDir().apply(config);
-            client.setCache(cache);
-        }
```

Comment:
@jglick Do you think worth reset the cache each time Jenkins receives a webhook? This is something happens (or should happen) very often.